sansa-ml-spark/pom.xml

```diff
@@ -132,6 +166,30 @@
 			<groupId>org.json</groupId>
 			<artifactId>json</artifactId>
 		</dependency>
+		
+		<!-- https://mvnrepository.com/artifact/org.datasyslab/geospark -->
+<dependency>
+    <groupId>org.datasyslab</groupId>
+    <artifactId>geospark</artifactId>
+    <version>1.1.3</version>
```

Comment:
I have added ${geospark.version} in properties section of pom file. While adding  geospark-sql.${spark.version}, its throwing an error. Since spark version in pom is 2.3.1 whereas this dependency is using 2.3 only. Is there any other way around?

Reply:
Sure. You could specify another property `<spark.binary.version>2.3</spark.binary.version>` which is only the major version of spark involved and we are sure that the spark version is aligned throughout all dependencies depending on spark versioning.